Source/Csla.Blazor.WebAssembly/State/SessionManager.cs

```diff
@@ -41,132 +41,134 @@ public Session GetCachedSession()
       return _session;
     }
 
-        /// <summary>
-        /// Retrieves the current user's session from
-        /// the web server to the wasm client
-        /// if SyncContextWithServer is true.
-        /// </summary>
-        /// <param name="timeout">The timeout duration for the operation.</param>
-        /// <returns>The retrieved session.</returns>
-        public async Task<Session> RetrieveSession(TimeSpan timeout)
+    /// <summary>
+    /// Retrieves the current user's session from
+    /// the web server to the wasm client
+    /// if SyncContextWithServer is true.
+    /// </summary>
+    /// <param name="timeout">The timeout duration for the operation.</param>
+    /// <returns>The retrieved session.</returns>
+    /// <exception cref="TimeoutException">Thrown when the specified timeout is exceeded.</exception>
+    public async Task<Session> RetrieveSession(TimeSpan timeout)
+    {
+        try
```

Comment:
Wrong formatting. 3 spaces instead of 2 within the methods.
Looks like this is true for the whole file.

Reply:
we could add this on all of the .csproj files  @StefanOssendorf 


```xml
<PropertyGroup>
  <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
</PropertyGroup>
```